plugins/aep-optimize-prerequisites/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Is this one a duplicate of  `plugins/aep-optimize-peer-dependencies/index.ts`?

Reply:
@amadt  Peer Dependencies makes sure you have offers or target installed, this one makes sure you have edge and optimize set up. I ended up splitting them up to give better feedback in the dashboard, but I have no issue combining them back together if you think that'd be good.

Reply:
Nope. That seems fine